packages/neocmakelsp/package.yaml

```diff
@@ -10,7 +10,18 @@ categories:
   - LSP
 
 source:
-  id: pkg:cargo/neocmakelsp@0.7.9
+  id: pkg:github/Decodetalkers/neocmakelsp@v0.7.9
```

Comment:
The repo name changes to `github/neocmakelsp/neocmakelsp`

Reply:
The mason maintainer doesn't seem to care about this PR so I stop pursuing, I can close it and let you take over if you want.

Reply:
> The mason maintainer doesn't seem to care about this PR so I stop pursuing, I can close it and let you take over if you want.

Sure. I would love to.